### DISCUSSION TITLE
Linkedin申請第一步就卡住

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#300] [Q&A] [answered]
老師給我們作為期中書面報告參考的「AI私人教練讓你成為籃球高手」的網址
https://www.crunchbase.com/organization/nex-team-homecourt
點進去後，如果想看**Employee Profiles**，點進去會發現如下面這樣，
![image](https://user-images.githubusercontent.com/86505454/162143566-eb40ef83-8665-4271-8643-fa85f6a128c0.png)

每個人右邊(如David Lee的右邊)都有
![LinkedIn商標](https://user-images.githubusercontent.com/86505454/162141897-5c5a33e4-6107-44b4-8158-da5cf09280b8.png)
的圖案。點進去(用滑鼠「左邊的按鍵(左鍵)」才能點進去)，會發現這是LinkedIn的網站。因為我不是LinkedIn的會員(應該是免費的吧？)，因此我選擇使用Google帳號登入，並且設定了密碼。接下來就開始要填資料，但第一步我就卡住了，如下，
![image](https://user-images.githubusercontent.com/86505454/162143721-87752134-738f-42b0-b6da-e83a2fe7ba52.png)

我不知道做什麼。我上網查LinkedIn有沒有教學，但只查到這個[註冊加入LinkedIn](https://www.linkedin.com/help/linkedin/answer/47163/-linkedin?lang=zh-hant)，我目前卡住的步驟應該是`4.依提示完成額外步驟。`也就是LinkedIn沒有教學的部份。所以想問老師或是有申請過的同學，該如何繼續呢？如果是中央大學的學生會有學生特別版或免費升級嗎？謝謝。

**Comment by chia-kai-chang** [comment, marked as answer]
Linkedin是在職工作者的Facebook，用來曬個人學經歷的

**Reply by enoch-lightning** [reply]
原來Linkedin是這種東西，謝謝老師。我發現我沒有填
![image](https://user-images.githubusercontent.com/86505454/162437866-95272a05-17a7-45d0-bf00-e67b5e86d878.png)
也是可以看Linkedin中別人的學歷與經歷。原本從下面的email中點選「立即開始」
![image](https://user-images.githubusercontent.com/86505454/162438240-3f63af79-3bd8-4673-a01b-54f636bfb80c.png)
會出現
![image](https://user-images.githubusercontent.com/86505454/162437866-95272a05-17a7-45d0-bf00-e67b5e86d878.png)
現在點立即開始就會進到下面這個網頁
![image](https://user-images.githubusercontent.com/86505454/162438375-a37f53c7-7546-43e6-8769-0cf97c1fc382.png)
不知道為什麼會這樣。
想請問老師有設定下面這個或是別的東西嗎？
![image](https://user-images.githubusercontent.com/86505454/162437866-95272a05-17a7-45d0-bf00-e67b5e86d878.png)
謝謝您。
🦃
